src/policy/lockfreeimmortalspace.rs

```diff
@@ -50,6 +50,15 @@ impl<VM: VMBinding> SFT for LockFreeImmortalSpace<VM> {
     fn is_live(&self, _object: ObjectReference) -> bool {
         unimplemented!()
     }
+    fn pin_object(&self, _object: ObjectReference) -> bool {
+        unimplemented!()
```

Comment:
These methods for lock free immortal space should have the same behavior as immortal space.